src/assets/locales/de/main.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Catena-X SDE",
```

Comment:
This might be potentially wrong; Make sure this is really called Catena-X and not Eclipse Tractus-X